### DISCUSSION TITLE
Create new application

### DISCUSSION THREAD

**Discussion by LPeco** [frang75/nappgui_src#145] [General]
![error_creando_aplicacion](https://github.com/user-attachments/assets/b7056d79-49ae-4887-bfa3-64a567586d7d)
Hi, when I try to compile a new application, I get the error shown in the attached image with the command:
cmake --build build
It does not find the .h file, which I have installed in:
C:\nappgui\inc
Is it necessary to configure any environment variable or do I not have the SDK installed correctly?
Thank you very much for the help.

**Comment by frang75** [comment]
Please share your `CMakeLists.txt`

**Comment by LPeco** [comment]
Ok. Thanks.
[CMakeLists.txt](https://github.com/user-attachments/files/16506485/CMakeLists.txt)

**Comment by frang75** [comment]
There are two problems:
1) Typo error in your `CMakeLists.txt`
`target_link_libraries(naphello ${NAPPGUI_LIBRARIES})`

2) Issue in NAppGUI Windows install. Fixed in this commit https://github.com/frang75/nappgui_src/commit/6b0146cec6c78011e1ec8d3a49c5d01d32447bf7

Please update your repo, delete `C:\nappgui`, rebuild and reinstall.

**Comment by LPeco** [comment]
Perfect, now the project is generated. Sorry for my mistake.

Thank you very much.

**Reply by frang75** [reply]
Instead is possible build an application using CMake `add_executable()` directly, is better to use `NAppProject.cmake` high level macros.
https://nappgui.com/en/guide/newprj.html#h2

**Comment by LPeco** [comment]
Ok, I'll try it, thank you very much.

El mar, 6 ago 2024 a las 19:30, Francisco García Collado (<
***@***.***>) escribió:

> Instead is possible build an application using CMake add_executable()
> directly, is better to use NAppProject.cmake high level macros.
> https://nappgui.com/en/guide/newprj.html#h2
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/frang75/nappgui_src/discussions/145#discussioncomment-10256415>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AJNHJU22BQDXBYS7DG76BWTZQEB2PAVCNFSM6AAAAABMAJVK2WVHI2DSMVQWIX3LMV43URDJONRXK43TNFXW4Q3PNVWWK3TUHMYTAMRVGY2DCNI>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 
LAP
Burgos - España
